ivy/functional/backends/tensorflow/experimental/elementwise.py

```diff
@@ -307,15 +306,26 @@ def diff(
 
 @with_unsupported_dtypes({"2.9.1 and below": ("bfloat16, float16,")}, backend_version)
 def zeta(
-    x: Union[tf.Tensor, tf.Variable],
-    q: Union[tf.Tensor, tf.Variable],
-    /,
-    *,
-    out: Optional[Union[tf.Tensor, tf.Variable]] = None,
+        x: Union[tf.Tensor, tf.Variable],
+        q: Union[tf.Tensor, tf.Variable],
+        /,
+        *,
+        out: Optional[Union[tf.Tensor, tf.Variable]] = None,
 ) -> Union[tf.Tensor, tf.Variable]:
     return tf.math.zeta(x, q)
 
 
+@with_unsupported_dtypes({"2.9.1 and below": ("uint8", "uint16", "uint32", "uint64", "bfloat16", "int32")}, backend_version)
+def angle(
+        input: Union[tf.Tensor, tf.Variable],
+        /,
+        *,
+        deg: Optional[bool] = None,
+        out: Optional[Union[tf.Tensor, tf.Variable]] = None,
+) -> Union[tf.Tensor, tf.Variable]:
+    return tf.math.angle(input, name=None)
```

Comment:
I don't see the `deg` parameter being handled here, you should have a check if `deg` is True then multiply the function output by 180/pi.